src/Tizen.Applications.PackageManager/Tizen.Applications/PackageManager.cs

```diff
@@ -72,6 +72,7 @@ public static class PackageManager
         /// <summary>
         /// InstallProgressChanged event. This event occurs when a package is getting installed and the progress of the request to the package manager is changed.
         /// </summary>
+        /// <exception cref="IOException">Thrown when failed to subscribe package manager event.</exception>
```

Comment:
Please rephrase all of the new `<exception` lines: Thrown when subscribing to package manager event failed.

Reply:
Done. Thanks.